lib/rules/require-quoted-keys-in-objects.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Requires quoted keys in objects.
+ *
+ * Type: `Boolean`
+ *
+ * Values:
+ *
+ *  - `true` for strict mode
+ *
+ * #### Example
+ *
+ * ```js
+ * "requiresQuotedKeysInObjects": true
+ * ```
+ *
+ * ##### Valid for mode `true`
```

Comment:
The examples are overkill? If there's no objection to being verbose, I'd prefer to leave them in.